example/generator/SinewaveLimitProvider.js

```diff
@@ -94,32 +110,66 @@ define([
     };
 
     SinewaveLimitProvider.prototype.getLimits = function (domainObject) {
+
         return {
             limits: function () {
-                return {
+                return Promise.resolve({
+                    WATCH: {
+                        low: {
+                            color: "cyan",
+                            sin: -CYAN.sin,
+                            cos: -CYAN.cos
+                        },
+                        high: {
+                            color: "cyan",
+                            ...CYAN
+                        }
+                    },
                     WARNING: {
                         low: {
-                            cssClass: "is-limit--lwr is-limit--yellow",
+                            color: "yellow",
```

Comment:
Curious what the reason is for switching to colors instead of css classes here. Are all colors supported? If not, are the supported colors documented somewhere? Remember this constitutes part of our public API.

Reply:
Done. See : https://github.com/nasa/openmct/pull/3917/files#diff-b161966f7254af9c9c5382e859495c69622843e4602a6bb4208e08134e660121